app/serializers/current_user_serializer.rb

```diff
@@ -329,4 +330,8 @@ def can_see_emails
   def include_can_see_emails?
     object.staff?
   end
+
+  def can_see_ip
```

Comment:
Same pattern as above - `include_can_see_ip?` would be handy to not include this for non-staff users